docusaurus.config.js

```diff
@@ -80,6 +80,18 @@ const config = {
         'Licensed by ComCODE with <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/" target="_blank" rel="noopener noreferrer">CC BY-NC-SA 4.0</a> for creative and educational use.',
     },
   },
+
+  // Global markdown config
+  // See: https://docusaurus.io/docs/api/docusaurus-config#markdown
+  markdown: {
+    format: "mdx",
+    mermaid: false,
+    mdx1Compat: {
+      comments: false,
+      admonitions: false,
+      headingIds: true,
```

Comment:
I know the migration guide said to keep this compat on until they come up with a replacement, but since we don't actually have any content yet I'd like to consider turning it off. We don't want to enable contributors to use a compat feature that will become obsolete at Facebook's earliest convenience.

Reply:
It didn't bother to highlight the line I picked, but to be clear: I mean `headingIds`.

Reply:
```suggestion
      headingIds: false,
```
Sure, I'm not opposed to changing this. I suppose it would be more future proof. @seanmakesgames thoughts?